about/logo/index.md

```diff
@@ -9,17 +9,18 @@ image:
   creditlink:
 ---
 
-The Stan name and logo are registered trademarks of
-[NumFOCUS](http://numfocus.org/) under the direction of the [Stan Governing Body](/about#stan-governing-body-sgb).
+The Stan name and logo are registered trademarks of [NumFOCUS](http://numfocus.org/)
+under the direction of the [Stan Governing Body](/about#stan-governing-body-sgb).
 
-Both may be freely used when referencing Stan, such as in a blog post or lecture
-notes, but they may not be used in the branding of commercial entities, such
-as software packages or paid courses, without the express license of the Stan
-leadership board.  If you have any questions or would like to inquire about
-licensing the Stan name and/or logo for a commercial venture then please contact
-NumFOCUS.
+Both may be freely used when referencing Stan, such as in blog posts, lecture notes,
+or opensourced software packages, but they may not be used in the branding of commercial entities,
+such as paid software packages or paid courses, without the express license of the Stan leadership board.
+If you have any questions or would like to inquire about licensing the Stan name
+and/or logo for a commercial venture then please contact NumFOCUS.
```

Comment:
This needs to say how to contact NumFOCUS.

Reply:
```suggestion
and/or logo for a commercial venture then please contact NumFOCUS at <mailto:admin@numfocus.org>.
```